owrx/config/defaults.py

```diff
@@ -147,6 +147,7 @@
     squelch_auto_margin=10,
     google_maps_api_key="",
     map_position_retention_time=2 * 60 * 60,
+    callsign_url="https://www.qrzcq.com/call/{}",
```

Comment:
Personally, I'm not a big fan of any of the ham radio profile pages, so I'd prefer the default to be empty. I don't remember any previous requests for this particular feature, so I have no idea what the general opinion on this would be...

Reply:
The QRZCQ has been selected since it always shows at least the name and location of the operator. Another option would be QRZ, but it usually contains less public data. You are welcome to remove the default value completely or even insert "I AM NOT A BIG FAN" instead.

Reply:
You don't seem to be a big fan of discussions. Noted.